pyouroboros/dockerclient.py

```diff
@@ -237,6 +237,42 @@ def update(self):
 
                 created = self.client.api.create_container(**new_config)
                 new_container = self.client.containers.get(created.get("Id"))
+
+                # disconnect new container from old networks (with possible
+                # broken config)
+                for network in new_container.attrs['NetworkSettings']['Networks']:
+                    self.client.api.disconnect_container_from_network(
+                        container=new_container.attrs['Id'],
+                        net_id=network,
+                        force=True
+                    )
+
+                # connect the new container to all networks of the old container
+                for network in container.attrs['NetworkSettings']['Networks']:
+                    host_net = self.client.networks.get(network_id=network)
+                    if ('Options' in host_net.attrs
```

Comment:
can you make this `host_net.attrs.get(‘Options’, None)` for keeping with our coding style?